docs/en/operation/Admin-CLI.md

```diff
@@ -98,6 +98,22 @@ BlockInfo{id=16793993216, length=6, locations=[BlockLocation{workerId=8265394007
 This block belongs to file {id=16810770431, path=/test2}
 ```
 
+### metrics
+
+The `metrics` command provides operations for Alluxio metrics system.
+
+`metrics clear` will clear all the metrics stored in Alluxio leading master.
+This command is useful when getting metrics information in short-term testing.
+It should be used sparingly as it may affect the current metrics recording and cause some metrics bias.
+
+If `--all` option is used, all the metrics stored in Alluxio leading master
+and active workers will be cleared.
+
+```console
+$ ./bin/alluxio fsadmin metrics clear
+$ ./bin/alluxio fsadmin metrics clear --all
```

Comment:
Is there a scenario we would want to clear all the workers and not the master? Right now, the master is always cleared, and the flag has the ability to additionally clear the workers.

Reply:
If a user wants to clear all the worker metrics, he has a large probability to clear the master metrics as well.

In some cases, a user may only want to clear the metrics of a specific worker and gain information about this worker.

Reply:
Changed to clear the whole cluster by default,
if --master is passed in, clear metrics of leading master only
if --workers is passed in, clear metrics of alluxio workers
if --node <worker_hostnames_separated_by_comma> is passed in, clear metrics of specified workers